OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -388,10 +425,18 @@ ActorInfo ChooseBuildingToBuild(ProductionQueue queue)
 
 				foreach (var cell in cells)
 				{
-					if (!world.CanPlaceBuilding(cell, actorInfo, bi, null))
+					if (!world.CanPlaceBuilding(cell, actorInfo, buildingInfo, null))
+						continue;
+
+					// Don't clutter the base
```

Comment:
Can you remove this comment and the `// Don't block off resources` below? Imho your method names are already quite descriptive, so there is nothing gained by those comments.